### ISSUE TITLE
Enum and export (false negatives)

### ISSUE THREAD

**Issue by scls19fr** [tonyhffong/Lint.jl#236]
Hello,

When running Lint with the following code:

```julia
@enum ActionWhenLong shorten=1 multiple=2
export shorten, multiple
```

it raises

``
`
W361 multiple: exporting undefined symbol
```

I only find https://github.com/tonyhffong/Lint.jl/issues/94 related to enum... but that's a bit different here.

Kind regards
